packages/sdks/src/types/api-version.ts

```diff
@@ -1,2 +1,2 @@
 export type ApiVersion = 'v2' | 'v3';
-export const DEFAULT_API_VERSION: ApiVersion = 'v3';
+export const DEFAULT_API_VERSION: ApiVersion = 'v2';
```

Comment:
The new SDKs are still on `0.x` versions and have not reached `1.0`. For them, we would want to stick with `v3` as a default because we don't want to do a major version bump yet (they are still beta SDKs and we are not ready to bump to `v1.0`)